### DISCUSSION TITLE
Addition of game visual function

### DISCUSSION THREAD

**Discussion by pyunjiji** [seerge/g-helper#1361] [Ideas]
Hello. Do you have any thoughts of adding the game visual feature installed in the armoury?
My laptop is 2022 a17.
Originally, my laptop had a panel of fhd, but since I switched to qhd, the game visual function of amoury doesn't work.
I've seen answers to colors in frequently asked questions, but sometimes I have to change them (such as when I have to play a game that requires a clear distinction...)
Can you add a function? Of course, even if you add it, my laptop is not working in the armory, so it may not work in the g helper either.

**Comment by seerge** [comment]
Added in latest release https://github.com/seerge/g-helper/releases

**Reply by pyunjiji** [reply]
thanks!
But I don't have 'GameVisual' so it doesn't work.

**Reply by XynoCrest** [reply]
Is there a shortcut to switch between game visual modes, without going to the app? The default used to be FN + v, but pressing that doesn't seem to do anything.

**Comment by Draebor** [comment]
can confirm working on my 2021 tuf F17 FX706HC.

can finally switch to different visual modes for different games - quite like running the FPS mode for my battle royale games.

cheers for this update!!!

**Reply by pyunjiji** [reply]
Hi, do you happen to have a gamevisual folder on your tuf? I don't have it so it doesn't work. If you don't mind, could you upload your gamevisual folder?

**Reply by Draebor** [reply]
im not sure where to find the gamevisual folder?

**Reply by Draebor** [reply]
hmm found it,  but the profiles look model specific ie they are all prefixed with my laptops model number.

**Reply by pyunjiji** [reply]
My a17 doesn't seem to work because I replaced the display panel.
